minio/api.py

```diff
@@ -795,6 +803,19 @@ def put_object(self, bucket_name, object_name, data, length,
         if length > MAX_MULTIPART_OBJECT_SIZE:
             raise InvalidArgumentError('Input content size is bigger '
                                        ' than allowed maximum of 5TiB.')
+
+        if (part_size * MAX_MULTIPART_COUNT) > length:
+            raise InvalidArgumentError('Part size '
```

Comment:
@y4m4 comment is not complete. Shouldn't the check be `if length > (part_size * MAX_MULTIPART_COUNT):`

Reply:
no length is already checked @kannappanr